manuscript/draft.Rmd

```diff
@@ -640,42 +633,37 @@ the habitats and species being surveyed [@sauer1994]. Accounting for differences
 in observers reduced the average error in our point estimates and also improved
 the coverage of the confidence intervals. In addition, controlling for observer
 effects resulted in changes in which models performed best, most notably
-reducing the relative performance of the naive model's point estimates. This
-suggests that, prior to accounting for observer effects, the naive model
-performed well because it was capable of accommodating rapid shifts in estimated
-richness introduced by changes in the observer. These kinds of rapid changes
-were difficult for the other single-site models to accommodate, and so the
-average performance of the ARIMA and average models improved once this source of
-observation error was addressed. This demonstrates that modeling observation
-error can be important for properly estimating and reducing uncertainty in
-forecasts and can also lead to changes in the best methods for forecasting [Box
```

Comment:
I like this sentence in that it ties back in to the overarching theme of the ms and reminds readers that this particular method is important to the overall goals of better forecasting. What's your thinking on the reason to take it out?

Reply:
I honestly don't know why removed it. Putting it back in now..